src/components/marginals/Footer/Footer.jsx

```diff
@@ -0,0 +1,95 @@
+/* eslint-disable max-len */
+import React from 'react';
+import { Link } from 'gatsby';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { library } from '@fortawesome/fontawesome-svg-core';
+import {
+  faTwitterSquare,
+  faInstagram,
+  faLinkedin,
+  faGithubSquare,
+  faDiscord,
+} from '@fortawesome/free-brands-svg-icons';
+
+import {
+  Grid,
+  SectionLogo,
+  SectionLogoImg,
+  SectionLogoText,
+  SectionIcon,
+  SectionIconText,
+  SectionText,
+  SectionOrganiser,
+  SectionCard1,
+  SectionCard2,
+  SectionIcons,
+} from './Footer.styles';
+
+import Heading4 from '../../shared/Typography/Heading4';
+import FooterCard from './FooterCard';
+import Body from '../../shared/Typography/Body';
+import { OrganiserData, SocialLogos } from './Data';
+import { SectionContainer } from '../../shared';
+
+library.add(faInstagram, faTwitterSquare, faDiscord, faLinkedin, faGithubSquare);
+
+function Footer() {
+  return (
+    <SectionContainer>
+      <Grid>
+        <SectionLogo>
+          <SectionLogoImg>
+            <img
+              src='https://res.cloudinary.com/dalqfvowk/image/upload/project-oregano/assets/hkmogkgvx6kpf07xbln8.png'
+              alt=''
+            />
+          </SectionLogoImg>
+          <SectionLogoText>
+            <Heading4>HackNITR 3.0</Heading4>
+          </SectionLogoText>
+        </SectionLogo>
+        <SectionIcon>
+          <SectionIcons>
+            {SocialLogos.map(({ icon, link }) => (
+              <Link key={icon} href={link}>
+                <FontAwesomeIcon icon={icon} opacity='40' color='#ffffff' size='2x' />
+              </Link>
+            ))}
+          </SectionIcons>
+          <SectionIconText>
+            <p className='text-right'>
+              <Body>logo Lorem ipsum dolor sit amet consectetur adipisicing.</Body>
+            </p>
+          </SectionIconText>
+        </SectionIcon>
+        <SectionCard1>
+          <FooterCard
+            img='https://res.cloudinary.com/dalqfvowk/image/upload/project-oregano/assets/j1hylhwi5lv6kyaksvbj.png'
+            Title='About NIT Rourkela'
+            Text='Sit mi, porttitor sapien, morbi id. Arcu, nibh facilisis
+           ac nec. In nec pulvinar praesent neque eget velit nunc sit potenti. Diam dignissim.'
+          />
+        </SectionCard1>
+        <SectionCard2>
```

Comment:
@ORKO06 tell me one thing. Both the cards in the footer are of same structure and style. So, what's the need of making two card components i.e `SectionCard1` and `SectionCard2`?

Don't you think it's redundant?